### PR TITLE
[Bug](security) BE download_files function exists log print sensitive msg #28592

### LINKED ISSUE #28592: [Bug] BE download_files function exists log print sensitive msg
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Version

master
2.0.2

### What's Wrong?

image
download_files function exists print sensitive token msg

### What You Expected?

sensitive message need to be mask print

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by LuGuangming**
## Proposed changes

Issue Number: close #28592

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by yiguolei — DISMISSED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Comment by LemonLiTree** [comment]
please format code
https://doris.apache.org/community/developer-guide/cpp-format-code/

File: be/src/olap/task/engine_clone_task.cpp [hunk 1]
```diff
@@ -827,4 +828,9 @@ Status EngineCloneTask::_finish_full_clone(Tablet* tablet,
     // TODO(plat1ko): write cooldown meta to remote if this replica is cooldown replica
 }
 
+std::string _mask_token(const std::string& str) {
```
Comment (rohitrs1983) [inline be/src/olap/task/engine_clone_task.cpp:831]: class name should be added. EngineCloneTask::_mask_token

Comment (LuGuangming) [reply be/src/olap/task/engine_clone_task.cpp]: OK, fixed

**Comment by LemonLiTree** [comment]
run buildall

**Comment by yiguolei** [comment]
run buildall

**Review by rohitrs1983 — APPROVED** [review]

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**